docs/examples/aws-lambda.js

```diff
@@ -0,0 +1,10 @@
+const Telegraf = require('telegraf')
+const { makeHandler } = require('telegraf/serverless/aws-lambda')
```

Comment:
This import won't work: https://github.com/telegraf/telegraf/blob/fc5882cd52505aa81b7d0f8620838b9a6b4dad40/package.json#L16-L19